src/encoded/schemas/experiment_series.json

```diff
@@ -0,0 +1,211 @@
+            "title": "Biosample classification"
+        },
+        "biosample_ontology.organ_slims": {
+            "title": "Organ"
+        },
+        "biosample_ontology.term_name": {
+            "title": "Biosample term name"
+        },
+        "contributing_awards.project": {
+            "title": "Project"
+        },
+        "contributing_awards.rfa": {
+            "title": "RFA"
+        },
+        "contributors.title": {
+            "title": "Lab"
+        },
+        "date_released": {
+            "title": "Month released"
+        },
+        "organism.scientific_name": {
+            "title": "Organism"
+        },
+        "status": {
+            "title": "Status"
+        },
+        "target.investigated_as": {
+            "title": "Target category"
+        },
+        "target.label": {
+            "title": "Target of assay"
+        }
+    },
+    "columns": {
+        "accession": {
+            "title": "Accession"
+        },
+        "assay_term_name": {
+            "title": "Assay name"
+        },
+        "assay_title": {
+            "title": "Assay title"
+        },
+        "biosample_ontology.term_name": {
+            "title": "Biosample term name"
+        },
+        "contributors.title": {
+            "title": "Lab"
+        },
+        "contributing_awards.project": {
+            "title": "Project"
+        },
+        "organism.scientific_name": {
+            "title": "Organism"
+        },
+        "related_datasets.replicates.library.biosample.age_display": {
+            "title": "Biosample age",
+            "type": "string"
+        },
+        "related_datasets.replicates.library.biosample.life_stage": {
+            "title": "Life stage",
+            "type": "array"
+        },
+        "status": {
+            "title": "Status"
+        },
+        "target.genes.symbol": {
+            "title": "Target gene symbol"
+        },
+        "target.label": {
+            "title": "Target of assay"
+        }
+    },
+    "boost_values": {
+        "accession": 1.0,
+        "aliases": 1.0,
+        "assay_term_name": 1.0,
+        "biosample_ontology.term_id": 1.0,
+        "biosample_ontology.term_name": 1.0,
+        "biosample_ontology.classification": 1.0,
+        "biosample_ontology.organ_slims": 1.0,
+        "biosample_ontology.cell_slims": 1.0,
+        "biosample_ontology.developmental_slims": 1.0,
+        "biosample_ontology.synonyms": 1.0,
+        "contributing_awards.title": 1.0,
+        "contributing_awards.project": 1.0,
+        "contributing_awards.name": 1.0,
+        "contributors.institute_name": 1.0,
+        "contributors.institute_label": 1.0,
+        "contributors.title": 1.0,
+        "notes": 1.0,
+        "organism.name": 1.0,
+        "organism.scientific_name": 1.0,
+        "organism.taxon_id": 1.0,
+        "contributors.institute_name": 1.0,
+        "contributors.institute_label": 1.0,
+        "contributors.title": 1.0,
```

Comment:
These three "contributors" lines repeat a few lines above

Reply:
Repeat?

Reply:
"contributors.institute_name": 1.0, and the next two lines are already there a few lines above. VSCode marks these.

Reply:
yes they are in there twice.

Reply:
Sorry. Saw that. Thanks.